### PR TITLE
Update README.rst

### PR DESCRIPTION
**Pull request by ben-en**
Add local usage description for forum question.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -59,3 +59,22 @@ use::
...
+
+  {
+      "date": "2015-03-17",
+      "handle": "BreakingNews",
+      "id": "577895032209960960",
+      "text": "RT @breakingpol: Illinois Rep. Aaron Schock is resigning his seat in Congress - @politico http://t.co/sgCkJiSSyP",
+      "time": "18:11:27"
+  }
+
+Images may be provided and must be named ``$tweet-id``.ext. For example, if i had an image for the above tweet, it would
```
Comment (foxbunny) [inline README.rst]: Should explain how to obtain `$tweet-id` and provide an example using the JSON above.

EDIT: Sry, did not see the example in the next line. Example's fine. So just how to grab the `$tweet-id`.

Comment (ben-en) [inline README.rst]: $tweet-id is provided by twitter.

For example https://twitter.com/ThaneRichard/status/711941061971587073

Comment (foxbunny) [inline README.rst]: It does not say so in the document.

Comment (foxbunny) [inline README.rst]: Also, if twitter API knowledge is a requirement, it should be mentioned at the top of the section.

**Pull request merged**